builtin/providers/aws/resource_aws_redshift_subnet_group.go

```diff
@@ -24,25 +24,27 @@ func resourceAwsRedshiftSubnetGroup() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			"name": &schema.Schema{
+			"name": {
```

Comment:
Help me understand why `&schema.Schema` was removed.  Simply because it's implied by the type?